bin/create.js

```diff
@@ -57,6 +74,10 @@ var tokenOptions = {
         description: 'pre-compile a local file using the indicated library (for now only `babel` is supported and will read `.babelrc` if present)',
         type: 'string',
     },
+    auth: {
+      description: 'set webtask permisions',
```

Comment:
Maybe we can call it `access` or `acls`. The `auth` is confusing vs `auth0` sitting next to it.